src/prometheus_alert_rules/heartbeat.rule

```diff
@@ -0,0 +1,7 @@
+# Based on https://awesome-prometheus-alerts.grep.to/rules.html#prometheus-self-monitoring-1
+alert: Watchdog
```

Comment:
**suggestion:** Name this so a causal look at alertmanager ui immediately discloses what it's for.

```suggestion
alert: AlertmanagerSelfWatchdog
```

Reply:
I'm not sure `AlertmanagerSelfWatchdog` is any more clear. I don't think I would gain any info from that other than that it is a watchdog alert.